### DISCUSSION TITLE
blog/2020/10/25/chmod_modes

### DISCUSSION THREAD

**Discussion by giscus** [SuperKogito/SuperKogito.github.io#12] [Announcements]
# blog/2020/10/25/chmod_modes

convert chmod modes to octal and back to letters

https://superkogito.github.io/blog/2020/10/25/chmod_modes.html

**Comment by Maliuta** [comment]
Actually you should be using 4 octal digits to represent file/directory permissions. The first digit represents the suid/sgid/sticky state, then owner, group, all.

There is a good explanation [here] (https://www.redhat.com/sysadmin/suid-sgid-sticky-bit).
